### ISSUE TITLE
Blank Login Page

### ISSUE THREAD

**Issue by ralekeo** [volkanceylan/Serenity#2119]
Hi All, 

I am at my wits end over this having burnt 10 hours on it and could not find a working solution to this, serenity worked fine with Northwind Database, after adding my own tables and generating the pages using sergen successfully, the login page turned blank. 

Excluding the new folder created does not resolve it.

Serenity 2.9.21
TS is version 2.2.2
NPM 3.10.10
node 6.9.5

**Error in chrome console**
![image](https://cloud.githubusercontent.com/assets/26841943/24580923/4d19444c-1709-11e7-8fa7-1972e15d3b22.png)

**Warning List**

![image](https://cloud.githubusercontent.com/assets/26841943/24580964/296f6db8-170a-11e7-8795-beedf5868954.png)

**Comment by volkanceylan** [comment]
Why you named your entity identifier Dbo.Currency?? It's supposed to be a class name

**Comment by ralekeo** [comment]
Thank you.  Volkanceylan

I created the table user.dbo.currency then renamed it to user.currency after running sergen, 

However I observed that because I used the same database the classes were also created as a hidden file under Northwind and in a separate module which I specified.

These hidden files in NorthWind were the culprit. So excluding the newly created module did not return me to my previous state till I deleted the offending files


![image](https://cloud.githubusercontent.com/assets/26841943/24585244/9a0827ea-177d-11e7-922a-818a8e452884.png)
